.idea/runConfigurations/pytest_for_all__jax_.xml

```diff
@@ -1,48 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
Same here for this file: add it back and the contents should be: https://github.com/unifyai/ivy/blob/master/.idea/runConfigurations/pytest_for_all__jax_.xml